server/configuration.js

```diff
@@ -20,6 +22,11 @@ export const setEnvironment = () => {
             process.env.TRAY_ENDPOINT = productionGraphqlEndpoint;
             process.env.TRAY_APP_URL = appUrlProd;
             break;
+        case "eu1-prod":
```

Comment:
I noticed we're using double-quotes for strings here, but the rest of the file uses single quotes. As this repo is customer-facing we should probably be consistent